sklearn/preprocessing/tests/test_label.py

```diff
@@ -602,6 +603,14 @@ def test_label_binarize_multilabel():
 def test_invalid_input_label_binarize():
     with pytest.raises(ValueError):
         label_binarize([0, 2], classes=[0, 2], pos_label=0, neg_label=1)
+    assert_raises(ValueError, label_binarize, [0, 2], classes=[0, 2],
```

Comment:
Isn't this equivalent to the previous statement?

Reply:
it is, fixed it